core/pyrightconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "include": [
+        "src"
+    ],
+    "exclude": [
+        "src/apps/monero",
+        "src/typing.py",
+        "src/all_modules.py"
+    ],
+    "stubPath": "mocks/generated",
+    "typeCheckingMode": "basic",
```

Comment:
Unfortunately, yes. Typing coming from `loop` and `ui` is really weak, so it produces a lot of "unknown variable/argument/member" errors.

It would be nice to try to use pyright_tool to enable strict mode only for `src/apps` or something like that.